CloudFoundry/content/apps/static.md

```diff
@@ -0,0 +1,66 @@
+---
+menu:
+  main:
+    parent: apps
+title: Deploying Static Sites
+weight: 10
+---
+
+Create `index.html`:
+
+```
+$ touch index.html
+```
+
+Add some markup:
+
+```html
+<html>
+  <head>
+    <title>Static Site</title>
+  </head>
+  <body>
+    <p>Welcome to the static site!</p>
+  </body>
+</html>
+```
+
+Create a `manifest.yml` that uses the [`staticfile-buildpack`](https://github.com/cloudfoundry/staticfile-buildpack):
+
+```yml
+---
+applications:
+- name: my-static-site
+  memory: 64M
+  buildpack: https://github.com/cloudfoundry/staticfile-buildpack.git
```

Comment:
I'll re-write that a little to instruct people to create an empty Staticfile to use the provided buildpack